firmware/src/src/defines.h

```diff
@@ -26,7 +26,7 @@
 #define BT_PERIOD 12500          // (us) Bluetooth update rate
 #define SERIAL_PERIOD 30         // (ms) Serial processing
 #define DATA_PERIOD 2            // Multiplier of Serial Period (Live Data Transmission Speed)
-#define SENSOR_PERIOD 16666      // (us) 60hz Read Sensors
+#define SENSOR_PERIOD 4000       // (us) 100hz Read Sensors
```

Comment:
hmm, if the earlier one 16.666 msec was 60hz, then 4000 should be more like 250hz.  is the system clock not actually 1mhz?

Reply:
Didn't update the comments either.. I will just remove them, no point in keeping them updated when the value is right there. Will keep the units.

I should rename that define too, it's actually a sleep time there. Because the IC2 reads take some time it works out to on average 166Hz. Probably should do like I did in Calculate and adjust the sleep period based on the loop time.

I'll add that.